platform/nephos/sonic-platform-modules-cig/cs6436-56p/modules/x86-64-cig-cs6436-56p-sfp.c

```diff
@@ -0,0 +1,1454 @@
+/*

```

Comment:
please use optoe driver instead of your own sfp module driver.

Reply:
As we use CPLD to control the SFP and QSFP module in the hardware design, we can't access it by the optoe driver.